src/dreamster.cpp

```diff
@@ -66,12 +44,76 @@ void Dreamster::move_motor(int p, int n, int16_t speed)
   }
 }
 
-uint16_t Dreamster::scan_sensor(int trigger, int echo)
+int Dreamster::scan_sensor(int trigger, int echo)
```

Comment:
why return a long on a int función? also, the cast from float to int is happening but you removed the explicit cast. why?